src/components/NewCommentForm.tsx

```diff
@@ -1,8 +1,81 @@
-import React from 'react';
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import { CommentData } from '../types/Comment';
+
+type Props = {
+  onAddComment: (commnet: CommentData) => Promise<void>;
+  postId: number,
+};
+
+export const NewCommentForm: React.FC<Props> = ({ onAddComment, postId }) => {
+  const [nameInput, setNameInput] = useState('');
+  const [emailInput, setEmailInput] = useState('');
+  const [commentInput, setCommentInput] = useState('');
```

Comment:
```suggestion
   сonst [comment, setComment] = useState({
      name: '',
      email: '',
      body:'',
    });
```
let's combine it into one use state